### DISCUSSION TITLE
Domino/Mail template System requirements

### DISCUSSION THREAD

**Discussion by bestinformatica** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#120] [Q&A]
It seems there are no system requirements about Domino Server or mail template release, correct? Does DOMI work also with Domino 9.0.1 and a 9.0.1 Mail template? Of course Notes client is 11.0.1 FP3 or greater

thanks

**Comment by GulfCoach** [comment]
Hi,
I am sure it will work because only the Notes Client has to interpret the DOMI Code.
Yours Ulf
